src/pybrake/notifier.py

```diff
@@ -73,13 +72,19 @@ def __init__(
                 default value [re.compile("password"), re.compile("secret")]
         :param remote_config: Set up configuration from the Airbrake server,
                 default value False
+        :param max_failed_queue: Set up length of failed stats queue
```

Comment:
What do you think if we renamed this to `max_backlog_size`? Technically, unset metrics is a backlog waiting to be sent.